pkg/pmem-csi-operator/controller/deployment/controller_driver.go

```diff
@@ -37,6 +43,25 @@ const (
 	nodeControllerPort    = 10001
 )
 
+// A list of all object types potentially created by the operator,
+// in this or any previous release. In other words, this list may grow,
+// but never shrink, because a newer release needs to delete objects
+// created by an older release.
+// This list also should keep in sync with the operator RBAC rules.
```

Comment:
s/should keep/must be kept/

Add the same comment in the declaration of the RBAC rules, please.

Reply:
Fixed.